packages/eslint-config-node/index.js

```diff
@@ -178,5 +178,10 @@ module.exports = {
         unusedExports: true,
       }],
     },
+  }, {
+    files: ['*config.js'],
```

Comment:
We already have an override for config files [here](https://github.com/strvcom/code-quality-tools/blob/master/packages/eslint-config-base/index.js#L681), I'd rather update the file globs there than create a new section.

While you are at it, I think it might be a good idea to also add a glob for some configuration files commonly found in the root of the project:

```
/*.config.js
```

Also, not sure if that pattern (`*config.js`) should also be limited to the root directory of the project? And secondly, should it not also include a variant with capical `C` (`*Config.js`)? What program actually uses this form? It would have to be ie. `prettierconfig.js` or `prettier-config.js`. 🤔 I think this is too loose, especially if it is not limited to the project's root.

Reply:
Personally, I am against having config just in the root of a project. On frontend it happens quite often that you have a config for API and config for Client. Both usually stored somewhere in `src` folder.

Reply:
I would go with `'*[cC]onfig.js'` (and `'*[cC]onfig.[jt]s'` in Typescript) to cover all cases. Especially avoiding the `/*.` part which would limit this to root of the project and only check files prefixed with another word before `config`. In our case we have files such as `src/config.ts` and `src/store/config.ts` or similar which wouldn't be cover by he pattern oherwise.

Reply:
The whole point of having this rule enabled is that accessing stuff on `process.env` is expensive because Node needs to do a C system call. This is especially problematic on backend as it might slow it down if you need to access several env vars for every request.

So, the goal of this rule and this exception was to convince people to have a single file that would collect all the env vars the program needs and export it as a regular module that could be used all over the codebase with no performance penalty.

Is that not acceptable on frontend as well? Can you not have a single _environment.ts_ module or something where you would turn this rule off, collect all the env vars and export them as a module?